src/kusto/azext_kusto/generated/_params.py

```diff
@@ -39,126 +39,130 @@ def load_arguments(self, _):
         c.argument('cluster_resource_id', help='Resource id of the cluster that follows a database owned by this cluste'
                    'r.')
         c.argument('attached_database_configuration_name', help='Resource name of the attached database configuration i'
                    'n the follower cluster.')
 
     with self.argument_context('kusto cluster diagnose-virtual-network') as c:
         c.argument('resource_group_name', resource_group_name_type)
-        c.argument('cluster_name', help='The name of the Kusto cluster.')
+        c.argument('cluster_name', options_list=['--name', '-n'], help='The name of the Kusto cluster.',
+                   id_part='name')
 
     with self.argument_context('kusto cluster list-follower-database') as c:
         c.argument('resource_group_name', resource_group_name_type)
-        c.argument('cluster_name', help='The name of the Kusto cluster.')
+        c.argument('cluster_name', options_list=['--name', '-n'], help='The name of the Kusto cluster.')
 
     with self.argument_context('kusto cluster list-language-extension') as c:
         c.argument('resource_group_name', resource_group_name_type)
-        c.argument('cluster_name', help='The name of the Kusto cluster.')
+        c.argument('cluster_name', options_list=['--name', '-n'], help='The name of the Kusto cluster.')
 
     with self.argument_context('kusto cluster list-sku') as c:
         c.argument('resource_group_name', resource_group_name_type)
-        c.argument('cluster_name', help='The name of the Kusto cluster.')
+        c.argument('cluster_name', options_list=['--name', '-n'], help='The name of the Kusto cluster.')
 
     with self.argument_context('kusto cluster remove-language-extension') as c:
         c.argument('resource_group_name', resource_group_name_type)
-        c.argument('cluster_name', help='The name of the Kusto cluster.')
-        c.argument('value', action=AddLanguageExtensionsValue, nargs='+', help='The list of language extensions. Expect'
-                   ' value: language-extension-name=xx.')
+        c.argument('cluster_name', options_list=['--name', '-n'], help='The name of the Kusto cluster.',
+                   id_part='name')
+        c.argument('value', action=AddClustersValue, nargs='+', help='The list of language extensions.')
 
     with self.argument_context('kusto cluster start') as c:
         c.argument('resource_group_name', resource_group_name_type)
-        c.argument('cluster_name', help='The name of the Kusto cluster.')
+        c.argument('cluster_name', options_list=['--name', '-n'], help='The name of the Kusto cluster.',
+                   id_part='name')
 
     with self.argument_context('kusto cluster stop') as c:
         c.argument('resource_group_name', resource_group_name_type)
-        c.argument('cluster_name', help='The name of the Kusto cluster.')
+        c.argument('cluster_name', options_list=['--name', '-n'], help='The name of the Kusto cluster.',

```

Comment:
This would be a breaking change, please confirm this change.

Reply:
I'll revert the change.

Reply:
I removed the "options_list" from all resources and updated the help file as well